File: aroundHubSpringBoot/src/main/java/com/example/aroungHubJavaSpringBoot/aroundHubSpringBoot/data/dao/ProductDAO.java
package com.example.aroungHubJavaSpringBoot.aroundHubSpringBoot.data.dao;

import com.example.aroungHubJavaSpringBoot.aroundHubSpringBoot.data.entity.ProductEntity;

public interface ProductDAO {
    ProductEntity saveProduct(ProductEntity productEntity);
    ProductEntity getProduct(String productId);
}

File: aroundHubKotlinSpringBoot/src/main/kotlin/com/example/aroungHubKotlinSpringBoot/aroundHubKotlinSpringBoot/data/dao/ProductDAO.kt
package com.example.aroungHubKotlinSpringBoot.aroundHubKotlinSpringBoot.data.dao

import com.example.aroungHubKotlinSpringBoot.aroundHubKotlinSpringBoot.data.entity.ProductEntity

interface ProductDAO {
    fun saveProduct(productEntity: ProductEntity) : ProductEntity
    fun getProduct(productId: String) : ProductEntity
}